docs/design/2023-06-08-import-into-statement.md

```diff
@@ -0,0 +1,162 @@
+# IMPORT INTO statement
+
+- Author: [D3Hunter](https://github.com/D3Hunter)
+- Tracking Issue: [#42930](https://github.com/pingcap/tidb/issues/42930)
+
+## Background and Benefits
+
+In order to reduce the complexity of deploying, configuring, and maintaining
+Lightning, and to enhance the data import experience (as many configuration
+options are not well-supported through the web UI). Especially when doing
+parallel import for large mount of data, it's easy to mis-config and cause
+the whole import fail.
+
+So we plan to introduce a new SQL statement called "IMPORT INTO" to integrate
+the functionality of the Lightning local backend. With this command, users can
+accomplish data import tasks that previously required the use of the Lightning
+local backend separately, thereby lowering the barrier for importing data.
+
+## Goal
+
+Add a new IMPORT INTO statement to support importing data into TiDB using lightning
+local backend. This statement will be run under [Distributed Execution Framework](./2023-04-11-dist-task.md).
+
+SQL syntax:
+```sql
+IMPORT INTO tbl_name [(col_name_or_user_var [, col_name_or_user_var] ...)]
+[SET col_name=expr [, col_name=expr] ...]
+FROM 'file_or_url'
+[FORMAT string]
+[WITH option=val [, option=val]...]
+```
+
+Supported data formats: CSV, SQL, PARQUET.
+
+`WITH` options which are only allowed for CSV format:
+- `character_set ='<string>'` to specify the character set of the file.
+- `fields_terminated_by ='<string>'` to specify the field delimiter.
+- `fields_enclosed_by ='<char>'` to specify the field enclosure character.
+- `fields_escaped_by ='<char>'` to specify the escape character.
+- `fields_defined_null_by ='<string>'` to specify the string that represents NULL.
+- `lines_terminated_by ='<string>'` to specify the line terminator.
+- `skip_rows =<number>` to specify the number of rows to skip at the beginning of the file.
+
+Other `WITH` options that's supported for all formats:
+- `detached` to run in detached mode.
+- `disk_quota`: see lightning `disk-quota`.
+- `checksum_table`: see lightning `checksum`.
+- `max_write_speed`: control how fast we write to TiKV.
+- `disable_tikv_import_mode`: to avoid switching TiKV into import mode.
+- `thread`: to control import concurrency.
+
+We also add two statements to show or cancel the import job:
+
+```sql
+SHOW IMPORT JOBS;
```

Comment:
Why not "ADMIN SHOW IMPORT JOBS"?

Reply:
unlike DDL, import job creator can use `show import jobs` too